src/app/list/list.page.ts

```diff
@@ -28,10 +28,15 @@ export class ListPage implements OnInit {
         icon: this.icons[Math.floor(Math.random() * this.icons.length)]
```

Comment:
This part assigns a random icon to each item.

Reply:
Cool